### ISSUE TITLE
look into dwarfs?

### ISSUE THREAD

**Issue by vsoch** [apptainer/singularity#5746]
Heyo! This popped into my radar recently, and if you look at the comparisons with squashfs (at the bottom of the README) it seems to be faster and produce smaller binaries. Maybe worth looking into? https://github.com/mhx/dwarfs

**Comment by dtrudg** [comment]
Hi @vsoch - thanks, that's an interesting project, but this is probably something that would be best to come back to in say, a year.

The main advantage of SquashFS is that we can assume it's available everywhere, and is well supported due to being heavily used by other projects including e.g. in distro live images.

Since dwarfs is something that's brand new (first release 16 days ago), and not heavily used, it's probably worth waiting to see if it stays as a maintained project over time before it's considered for Singularity. Erofs (which was merged into Linux mainline) would definitely be something to look at in parallel too, if we go down this route.

**Comment by vsoch** [comment]
Cool sounds good! It will be fun if/when you try it. :popcorn:
